packages/typespec-azure-resource-manager/test/rules/operation-usage.test.ts

```diff
@@ -87,39 +87,6 @@ describe("typespec-azure-resource-manager: detect non-post actions", () => {
         message: "Actions must be HTTP Post operations.",
       });
   });
-
-  it("Allows post actions for authorized provider actions", async () => {
```

Comment:
would maybe be good to keep this test now that you have it but check the opposite(still emit a warning there)

Reply:
I just saw this! let me add this :)